app/views/report_templates/jobs_invocations.erb

```diff
@@ -0,0 +1,39 @@
+<%#
+name: Jobs - Invocations
+snippet: false
+template_inputs:
+- name: job_id
+  required: true
+  input_type: user
+  description: ID of job to report
+  advanced: false
+  value_type: plain
+  resource_type: JobInvocation
+  hidden_value: false
+- name: hosts
+  required: false
+  input_type: user
+  description: Field for filter hosts of job invocation. Leave blank for all hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+  hidden_value: false
+model: ReportTemplate
+-%>
+<%- invocation = find_job_invocation_by_id(input('job_id')) -%>
+<%- input_hosts = input('hosts') -%>
+<%- input_hosts = '*' if input_hosts.blank? -%>
+<%- load_hosts(search: "#{input_hosts} and job_invocation.id = #{input('job_id')}").each do |batch| -%>
+<%-   batch.each do |host|  -%>
+<%-     task = invocation.sub_task_for_host(host) -%>
+<%-     task.action_output['outputs'].each do |output| -%>
```

Comment:
Why not?
```suggestion
<%-     task.main_action.live_output.each do |output| -%>
```

Reply:
I think that I understand what are trying for. However I think that `live_output` is not allowed for safemode, yet. It's not problem, I can add jail for `RunHostJob` but I think we will facing to similar problem like https://github.com/theforeman/foreman_remote_execution/pull/453#discussion_r488658734